flake.nix

```diff
@@ -15,7 +15,7 @@
   };
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.11";
+    nixpkgs.url = "github:mikatammi/nixpkgs/nvidia_x11_aarch64_nixos-unstable";
```

Comment:
This worries me a lot! Is there really no other way of doing this without forking nixpkgs?

Reply:
The Nvidia drivers work with unstable version but it doesn't work with aarch64 yet, unfortunately.

Reply:
Could we just wait a month before merging this, then the driver will be in the new nixos-23.05 stable branch. The other way around this would be to add the package through an overlay

Reply:
but the overlay is so lot of work so I'd just rather not do extra work but just wait for a month :)